bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gnirs/GNIRSParams.java

```diff
@@ -156,9 +156,11 @@ public double getValue() {
         SW_8("3.0 arcsec", 3.0, "3.0") {
             @Override public boolean isObsolete() { return true; }
         },
-        IFU("IFU", 4.8, "IFU") {
+        IFU("IFU", 3.15, "IFU") {
             @Override public boolean isObsolete() { return true; }
         },
+        LR_IFU("LR-IFU", 3.15, "LR-IFU"),
```

Comment:
Will this need a change on the PIT?

Reply:
Eventually, and in templates etc.  I think for now though they will just be tested.  At any rate, PIT changes weren't mentioned in the task.